### PR TITLE
Enable lvm full encrypt in s390x

### PR DESCRIPTION
**Pull request by jknphy**
Adapt test scenarios full-lvm-encrypt and lvm-encrypt-separate-boot with libyui in s390x (zKVM):
    - Add test data for full-lvm-encrypt in s390x
    - Handle setup/teardown of libyui in s390x (zKVM)

- Related ticket: [poo#78016](https://progress.opensuse.org/issues/78016)
- Job Group MR: [MR#321](https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/321)
- Verification run: [lvm-full-encrypt](https://openqa.suse.de/tests/5107421) | [lvm-encrypt-separate-boot](https://openqa.suse.de/tests/5107445)

### CODE REVIEW

File: lib/YuiRestClient.pm [hunk 1]
```diff
@@ -65,8 +71,14 @@ sub setup_libyui {
 }
 
 sub teardown_libyui {
-    assert_screen('startshell', timeout => 100);
-    type_string_slow "exit\n";
+    if (get_var('S390_ZKVM')) {
+        wait_serial('ATTENTION: Starting shell', 120) || die "start shell didn't show up";
+        save_svirt_pty;
```
Comment (rwx788) [inline lib/YuiRestClient.pm:76]: Do we need this line? pty should be already set when we finish the installation.

File: lib/YuiRestClient.pm [hunk 2]
```diff
@@ -54,9 +53,16 @@ sub connect_to_app {
 }
 
 sub process_start_shell {
-    assert_screen('startshell', timeout => 500);
-    type_string_slow "extend libyui-rest-api\n";
-    type_string_slow "exit\n";
+    if (get_var('S390_ZKVM')) {
+        wait_serial('ATTENTION: Starting shell', 120) || die "start shell didn't show up";
+        save_svirt_pty;
```
Comment (rwx788) [inline lib/YuiRestClient.pm:58]: Same question here, as start shell appears after we spawn the VM, so this call should be already done in bootloader, isn't it?

File: lib/YuiRestClient.pm (hunk 1 above)
Comment (jknphy) [reply lib/YuiRestClient.pm:76]: I was wondering the same, I will delete it.

File: lib/YuiRestClient.pm (hunk 2 above)
Comment (jknphy) [reply lib/YuiRestClient.pm]: very likely [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L788) and I also spot it in first screenshot in bootloader, good catch. Re-running verification: https://openqa.suse.de/tests/5107441

Comment (jknphy) [reply lib/YuiRestClient.pm]: It didnt' work, [ambiguous redirect error](I got https://openqa.suse.de/tests/5107441) I will keep then.

Comment (rwx788) [reply lib/YuiRestClient.pm:58]: That's strange, but doesn't harm to call it here too, we can address it later on. Cheers.

**Pull request merged**